API/src/routes/secret.json

```diff
@@ -0,0 +1 @@
+{"secret":  "X?N+C^NFXk@B(F0QD/fA;Agz8+ngKK"}
```

Comment:
lol whats this secret for btw?

Reply:
secret is used for authenticating the token in events.use and games.use. Have to find a way to hide it though, like as in not on github for everyone to see.

Reply:
Yes, please do that. For now, get a new secret since this one has already been pushed to Github. Don't push that new secret to Git, maybe just send it privately to @terbb since he is managing the server and maybe he can find a way to get it onto the server without publicly displaying it.